add-ons/fabric8/README.md

```diff
@@ -0,0 +1,50 @@
+# Fabric8 Addon
+
+This addon provides an easy way to install fabric8 on MiniShift. 
+
+Fabric8 provides a complete developer platform for creating projects, editing them and performing full CI / CD for the entire developer lifecycle.
+
+## Setup GitHub OAuth client ID and secret
+
+Fabric8 has GitHub integration letting you browse repositories, create new repositories, edit projects and setup automated CI / CD jobs with webhooks on github. 
+
+This requires an [OAuth application to be setup on your github account](https://developer.github.com/apps/building-integrations/setting-up-and-registering-oauth-apps/registering-oauth-apps/) for fabric8 and you need to obtain the client ID and secret for the OAuth application.
+
+
+You will need to register the redirect URI in the OAuth application to point to the output of this command:
+
+```
+echo http://keycloak-fabric8.$(minishift ip).nip.io/auth/realms/fabric8/broker/github/endpoint
+```
+
+So please follow the following steps using the above redirect URL and `http://fabric8.io` as a sample homepage URL:
+
+
+![Register OAuth App](./images/register-oauth.png)
```

Comment:
This image does not exist. Maybe we can add it to the pull request or we remove it. WDYT?